### PR TITLE
Rename workload KSA name to 'default'

### PR DESCRIPTION
**Pull request by askoriy**

### CODE REVIEW

**Review by Santos92 — APPROVED** [review]
LGTM

File: modules/workload-identity/vars.tf [hunk 1]
```diff
@@ -15,3 +15,9 @@ variable services {
   }))
   description = "List of services with IAM Roles to assign to the Services Service Account"
 }
+
+variable ksa_name {
+  description "The name of Kubernetes Service Account to bind workload for"
+  type = string
+  default = "default"
+}
```
Comment (anakovt) [inline modules/workload-identity/vars.tf:23]: new line

File: modules/workload-identity/vars.tf [hunk 2]
```diff
@@ -15,3 +15,9 @@ variable services {
   }))
   description = "List of services with IAM Roles to assign to the Services Service Account"
 }
+
+variable ksa_name {
+  description "The name of Kubernetes Service Account to bind workload for"
```
Comment (anakovt) [inline modules/workload-identity/vars.tf:20]: description =

File: modules/workload-identity/vars.tf [hunk 3]
```diff
@@ -15,3 +15,9 @@ variable services {
...
+  type = string
```
Comment (anakovt) [inline modules/workload-identity/vars.tf:21]: terraform fmt

**Pull request merged**